presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreContext.java

```diff
@@ -126,6 +132,11 @@ public Optional<String> getClientInfo()
         return clientInfo;
     }
 
+    public Set<String> getClientTags()
```

Comment:
On first review I didn't find a test for this new public method.

Reply:
It's a getter method. I didn't find other getter methods with unit test either. IMO, it would be redundant to do so

Reply:
It's a getter method.  I haven't noticed other similar methods being tested either. In my opinion, testing this would be unnecessary repetition.

Reply:
In my opinion, unit tests are there to test the behaviour of code, in a meaningful way, and getters/setters are only a means to an end. Writing unit tests for a getter method is proving to be a diversion for me.